src/pages/files/Share.tsx

```diff
@@ -72,15 +69,15 @@ export function Share(props: RouteComponentProps<MatchParams>): ReactElement {
 
   function onClose() {
     // POP means there is no history - nowhere to go back yet
```

Comment:
Just a minor thing, can you please remove this comment? It is a bit confusing now 🙂